.gitlab-ci.yml

```diff
@@ -292,3 +301,31 @@ publish-docs:
     - git push origin gh-pages --force
   after_script:
     - rm -rf .git/ ./*
+
+#### stage:                        fuzz
+
+fuzz-tests:
+    stage:                           fuzz
+    <<:                              *docker-env
+    variables:
+        # The QUICKCHECK_TESTS default is 100
+        QUICKCHECK_TESTS:            1000
+    rules:
+        - if: $CI_PIPELINE_SOURCE == "schedule"
+        - if: $CI_COMMIT_REF_NAME == "master"
+    script:
+        - cargo test --verbose --features ink-fuzz-tests --no-fail-fast --workspace -- fuzz_
+        - if [ $? -eq 0 ]; then exit 0; fi
+        - |
+             curl -X "POST" "https://api.github.com/repos/paritytech/ink/issues?state=all" \
+                -H "Cookie: logged_in=no" \
+                -H "Authorization: token ${GITHUB_TOKEN}" \
+                -H "Content-Type: text/plain; charset=utf-8" \
+                -d $'{
+                    "title": "Failing fuzz tests on master ('"$( date +"%d %b %Y" )"'),
+                    "body": "The job '"${CI_JOB_ID}"' just failed: '"${CI_JOB_URL}"'.",
```

Comment:
It would make sense to indicate the commit message with `CI_COMMIT_MESSAGE `. Or maybe deduct and link the GitHub or a branch of PR from which the git sha comes from.

Reply:
> and what's the default amount?

The comment states the default: `# The QUICKCHECK_TESTS default is 100`. Did you mean this?

Reply:
ah, I'm blind, thanks